Assets/BossRoom/Scripts/Shared/Game/UI/Editor/NetworkLatencyWarning.cs

```diff
@@ -25,20 +27,21 @@ void Update()
 
                 switch (chosenTransport)
                 {
-                    // adding this preprocessor directive check since LiteNetLib only injects latency in #DEBUG
-                    // todo MTT-1426 do this for UTP
-                    // #if DEBUG
-                    // case LiteNetLibTransport liteNetLibTransport:
-                    //     m_ArtificialLatencyEnabled = liteNetLibTransport.SimulatePacketLossChance > 0 ||
-                    //         liteNetLibTransport.SimulateMinLatency > 0 ||
-                    //         liteNetLibTransport.SimulateMaxLatency > 0;
-                    //     break;
-                    // #endif
                     case UNetTransport unetTransport:
                     case PhotonRealtimeTransport photonTransport:
-                    case UnityTransport UnityTransport:
                         m_ArtificialLatencyEnabled = false;
                         break;
+                    case UnityTransport unityTransport:
+// adding this preprocessor directive check since LiteNetLib only injects latency in #UNITY_EDITOR or in #DEVELOPMENT_BUILD
```

Comment:
The comment talks about LiteNetLib but actually we're dealing with UnityTransport here